.github/workflows/run_pr_benchmarks.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd $BRANCH_NAME/
+mkdir pr
+cd pr/
+git clone --branch $BRANCH_NAME --single-branch git@github.com:armughan11/NullAway.git
```

Comment:
Won't we need to clone this branch from whatever repository the pull request was created from?

Reply:
Fixed this, cloning directly from the repo now using Github REST API.